package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "cloud-slack-bot-example",
+  "version": "1.0.0",
+  "description": "A sample Slack Botkit bot.",
+  "main": "kittenbot.js",
+  "dependencies": {
+    "botkit": "0.1.1"
+  },
+  "devDependencies": {
+    "standard": "^3.0.0"
+  },
+  "scripts": {
+    "test": "standard"
```

Comment:
Do you mean `"test": "standard \"**/*.js\""`? When I used `"test": "standard \"**.*.js\""` it didn't catch any errors that I manually introduced.